evennia/commands/default/unloggedin.py

```diff
@@ -465,3 +465,64 @@ def func(self):
                 utils.get_evennia_version(),
             )
         )
+
+
+def _create_account(session, accountname, password, permissions, typeclass=None, email=None):
```

Comment:
This entire modification seems out of place in this PR? Besides, this looks like a duplication of the `Account.create` code ...